src/client.js

```diff
@@ -501,6 +501,11 @@ Client.prototype.more = function(data) {
 	};
 };
 
+Client.prototype.search = function(query) {
+	const messageStorage = this.messageStorage.find((s) => s.canProvideMessages());
+	return messageStorage.search(query);
```

Comment:
Need to handle the case when sqlite is disabled.

Reply:
We still need some kind of handling (and probably hiding the search UI) if sqlite is disabled OR if user has logs disabled.